Gemfile

```diff
@@ -54,7 +54,6 @@ group :gcs do
 end
 
 group :gae do
-  gem 'rubyzip'
 end
```

Comment:
RubyGems and Zlib are standard libraries (http://ruby-doc.org/stdlib-2.1.5/) you can expect to find in functional Ruby runtimes, so there is no need to add them to Gemfile.